tests/vendor/autoload.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
The vendor folder should not be committed.

Reply:
This file is referenced in S3Monitor.sh

Reply:
Then another way needs to be found. The vendor directory should never be committed. Whats the purpose of that script?

Reply:
The S3Monitor scripts monitor the S3 client for an incoming emails